### PR TITLE
fozzie-components@v1.6.0 - Package updates and eslint import helper added

### PR DESCRIPTION
**Pull request by ashleynolan**
### Added
- Added the `eslint-import-resolver-webpack` so that eslint can understand webpack special characters such as `@` used in import paths

### Changed
- Minor package updates

### CODE REVIEW

**Review by simonsandersje — DISMISSED** [review]

**Review by zlatinivanov-je — DISMISSED** [review]

**Comment by ashleynolan** [comment]
@simonsandersje @zlatinivanov-je Could I get a re-review. Had some linting issues I had to address.

**Comment by zlatinivanov-je** [comment]
Had a look but the build is still failing

**Comment by ashleynolan** [comment]
@zlatinivanov-je Have resolved build issieu. Ignore the Danger checks – simply as I'm not bumping the f-header/footer packages because it's a minor config change that can be rolled into the next release.

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,17 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v1.6.0
```
Comment (smilburn1990) [inline CHANGELOG.md:6]: NIT: Wrong date

File: packages/f-footer/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,13 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
+Latest (roll into next release)
+------------------------------
+*March 27, 2020*
```
Comment (smilburn1990) [inline packages/f-footer/CHANGELOG.md]: Same as above

File: packages/f-header/CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
+Latest (roll into next release)
+------------------------------
+*March 27, 2020*
```
Comment (smilburn1990) [inline packages/f-header/CHANGELOG.md]: Same as above

File: packages/f-vue-icons/CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
+v0.18.1
+------------------------------
+*March 30, 2020*
```
Comment (smilburn1990) [inline packages/f-vue-icons/CHANGELOG.md:9]: Same as above

**Review by smilburn1990 — CHANGES_REQUESTED** [review]

File: packages/f-vue-icons/CHANGELOG.md (hunk 4 above)
Comment (ashleynolan) [reply packages/f-vue-icons/CHANGELOG.md:9]: This one's actually right 😆

**Comment by ashleynolan** [comment]
@smilburn1990 Fixed those dates

**Review by smilburn1990 — APPROVED** [review]

**Review by DamianMullins — APPROVED** [review]

**Pull request merged**